redirect/routes.php

```diff
@@ -87,10 +87,10 @@ class Route {
             'description' => 'Mathematische Grundlagen1; Analysis',
             'moodle_id'   => '60'
         ],
-        'ffi_mathe1ü' => [
+        /*'ffi_mathe1ü' => [
```

Comment:
Don't Comment out used links, wenn links auf falsche kurse verweisen, ist das ein Moodle Problem, da die Kurze erneut verwendet werden, und nicht neu angelegt werden... Vgl `ffi_mg2`, welcher die selbe moodle_id hat
```suggestion
        'ffi_mathe1ü' => [
```